app/services/publish/public_cocina_service.rb

```diff
@@ -15,7 +15,9 @@ def initialize(cocina)
     # remove any file_set that doesn't have at least one published file
     # remove partOfProject (similar to how we remove tags from identityMetadata)
     def build
-      return cocina unless cocina.dro?
+      return cocina unless cocina.dro? || cocina.collection?
+
+      return cocina.new(administrative: build_administrative) if cocina.collection?
 
       cocina.new(structural: build_structural,
                  administrative: build_administrative)
```

Comment:
Given we have multiple checks in a couple different spots (lines 18 and 20), I wonder if we want to consider a different structure now, *e.g.*:

```suggestion
      if cocina_object.dro?
        cocina.new(structural: build_structural, administrative: build_administrative)
      elsif cocina_object.collection?
        cocina.new(administrative: build_administrative)
      else
        cocina
      end
```